OasisDEX/oasis-borrow
# Added RETH-A support

# [Added RETH-A support](https://app.shortcut.com/oazo-apps/story/6615/add-reth-product)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added RETH config for mainnet and goerli
- handled it on the product card level
  
## How to test 🧪
  <Please explain how to test your changes>

- test it on mainnet and goerli, RETH product card should be available for borrow and multiply in ETH assets list
- user should be able to open borrow / multiple vault using RETH
- user should be able to set up automations
